spark-testing/spark_utils.py

```diff
@@ -17,24 +17,11 @@
 DCOS_SPARK_TEST_JAR_PATH = os.getenv(DCOS_SPARK_TEST_JAR_PATH_ENV, None)
 DCOS_SPARK_TEST_JAR_URL_ENV = "DCOS_SPARK_TEST_JAR_URL"
 DCOS_SPARK_TEST_JAR_URL = os.getenv(DCOS_SPARK_TEST_JAR_URL_ENV, None)
-if not DCOS_SPARK_TEST_JAR_URL and not os.path.exists(DCOS_SPARK_TEST_JAR_PATH):
```

Comment:
should this be `or`?

Reply:
Looks like it.

Reply:
https://github.com/mesosphere/spark-build/pull/334/commits/f84786a385f8536c8cd2ca4d5b0b84bdd14cb2bf

Reply:
No, it shouldn't be an `or` ... I was confused because these are different variables.